prod/westeurope/support/functions_backoffice/web_test/terragrunt.hcl

```
@@ -0,0 +1,35 @@
+# Common
+dependency "application_insights" {
+  config_path = "../../../common/application_insights"
+}
+
+dependency "resource_group" {
+  config_path = "../../../common/resource_group"
+}
+
+dependency "function_app" {
+  config_path = "../function_app"
+}
+
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_application_insights_web_test?ref=v2.1.34"
+}
+
+inputs = {
+  name = "func-backoffice"
+
+  resource_group_name     = dependency.resource_group.outputs.resource_name
+  application_insights_id = dependency.application_insights.outputs.id
+  enabled                 = true
+  geo_locations           = ["emea-nl-ams-azr"]
+
+  url = format("https://%s/api/v1/info?code=%s",
+    dependency.function_app.outputs.default_hostname,
+    dependency.function_app.outputs.master_key)
```

```suggestion
  url = format("https://%s/api/v1/info", dependency.function_app.outputs.default_hostname)
```
Not expose master key. Wait to deploy this pr https://github.com/pagopa/io-functions-backoffice/pull/50

---

> Not expose master key. Wait to deploy this pr [pagopa/io-functions-backoffice#50](https://github.com/pagopa/io-functions-backoffice/pull/50)
@pasqualedevita   too late :-(    it's already live .... I'll delete it today.

---

Not worry we can rotate it :)

---

Fix deployed